code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -97,7 +97,9 @@
 	loc.visible_message("<span class='warning'>The corpse of [H.name] suddenly rises!</span>")
 
 /mob/living/simple_animal/hostile/blob/blobspore/death(gibbed)
-	..()
+	. = ..()
+	if(!.)
```

Comment:
this is...a bit confusing.

Reply:
This is checking if it successfully "died" from this call to `death`, and only executing the code below if it in fact died from this calling of `death`